model/pokemon.go

```diff
@@ -0,0 +1,6 @@
+package model
+
+type Pokemon struct {
```

Comment:
Please add a comment to an exported structure, interface, or function.

Reply:
Updated